packages/reframed/src/reframed.ts

```diff
@@ -15,7 +15,7 @@ export function reframed(
 ): {
 	iframe: HTMLIFrameElement;
 	container: HTMLElement;
-	ready: Promise<void>;
+	ready: Promise<[() => void, void]>;
```

Comment:
As discussed, could you add a TODO comment saying that this `ready` API should be polished? 🙏  (with optionally a gh issue if you want)

Reply:
(or better, the TODO comment/issue should state that this can be reverted back to `Promise<void>` once we handle the cleanup inside reframing itself (again as we discussed))